### ISSUE TITLE
Counters on home page not counting

### ISSUE THREAD

**Issue by samyue888** [ocupop/librestream#105]
Please trigger the count once the user gets to that section.

**Comment by paulstroot** [comment]
@samyue888 - This is ready for review at this URL.
https://librestreamdevelopcountup.gatsbyjs.io/

**Comment by samyue888** [comment]
@paulstroot That link is working correctly - thanks. How do we merge?

**Comment by paulstroot** [comment]
I'll merge this into the develop branch and then create a pull request.
Once the recaptcha is approved that pull request can be merged into "main"
to go live.

**Comment by samyue888** [comment]
Working now, thanks
